Get rid of the warning about not used return value from chdir.
diff -ru dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.orig/src/dionaea.c dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.new/src/dionaea.c
--- dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.orig/src/dionaea.c	2018-03-21 04:21:30.201354584 +0100
+++ dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.new/src/dionaea.c	2018-03-21 04:22:07.529846514 +0100
@@ -723,7 +723,10 @@
 			g_error("Could not chroot(\"%s\") (%s)", opt->root, strerror(errno));
 		} else
 		{
-			chdir("/");
+			if ( chdir("/") != 0 ) {
+				g_error("Could change to root directory '/' ... this shuoldn't happen exitting.\nErr: (%s)", strerror(errno));
+				return EXIT_FAILURE;
+			}
 		}
 	}
 
